src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -324,22 +324,40 @@ const PublicationReleasePage: NextPage<Props> = ({ data }) => {
                 Find and download files used in the production of this release.
               </p>
               <ul className="govuk-list govuk-!-width-full">
-                {data.downloadFiles.map(({ extension, name, path, size }) => (
-                  <li key={path}>
-                    <Link
-                      to={`${process.env.CONTENT_API_BASE_URL}/download/${path}`}
-                      analytics={{
-                        category: 'Downloads',
-                        action: `Release page ${name} file downloaded`,
-                        label: `File URL: /api/download/${path}`,
-                      }}
-                    >
-                      {name}
-                    </Link>
-
-                    {` (${extension}, ${size})`}
-                  </li>
-                ))}
+                {data.downloadFiles.map(
+                  ({ id: fileId, fileName, extension, name, size }) =>
+                    !fileId && name === 'All files' ? (
```

Comment:
Similar to previous

Reply:
I've made the same change as on the downloads page now, thanks.